pkg/api/scheme/scheme.go

```diff
@@ -29,11 +31,29 @@ import (
 	storagev1 "k8s.io/api/storage/v1"
 	storagev1beta1 "k8s.io/api/storage/v1beta1"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	apiregistrationv1 "k8s.io/kube-aggregator/pkg/apis/apiregistration/v1"
 	apiregistrationv1beta1 "k8s.io/kube-aggregator/pkg/apis/apiregistration/v1beta1"
 )
 
-var Scheme *runtime.Scheme
+type FilteredScheme struct {
+	*runtime.Scheme
+	excludedGVKs map[schema.GroupVersionKind]bool
```

Comment:
nit: I am a fan of the following because it uses less memory:
```suggestion
	excludedGVKs map[schema.GroupVersionKind]struct{}
```

Reply:
TIL